lab_6_pipeline/pipeline.py

```diff
@@ -165,22 +308,77 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        super().__init__(corpus_manager)
+        self._backup_analyzer = pymorphy2.MorphAnalyzer()
```

Comment:
and here you do not use names like stemmer, you see?

Reply:
yes, I see